src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -744,14 +744,17 @@ export const ComputeModalBase = ({
         [Utils.DEFAULT, () => runtimeTypes.gceVm] // for when there's no existing runtime
       )
 
+      const diskSize = Utils.cond([!!runtimeConfig?.diskSize, () => runtimeConfig.diskSize],
+        [!!runtimeConfig?.masterDiskSize, () => runtimeConfig.masterDiskSize],
+        [isDataproc(newRuntimeType), () => defaultDataprocMasterDiskSize],
+        [Utils.DEFAULT, () => defaultGceBootDiskSize])
```

Comment:
`Utils.DEFAULT` is used with `Utils.switchCase`. For a default case for `Utils.cond`, use a function instead of a `[predicate, function]` array.

```suggestion
        () => defaultGceBootDiskSize)
```